fedn/cli/run_cmd.py

```diff
@@ -70,28 +75,28 @@ def validate_cmd(ctx, path,input,output):
     dispatcher = Dispatcher(config, path)
     _ = dispatcher._get_or_create_python_env()
     dispatcher.run_cmd("validate {} {}".format(input, output))
-
-    # delete the virtualenv
-    if dispatcher.python_env_path:
-        logger.info(f"Removing virtualenv {dispatcher.python_env_path}")
-        shutil.rmtree(dispatcher.python_env_path)
+    if remove_venv:
```

Comment:
perhaps move this if statement to a separate function that can be used within each subcommand? A lot of redundancy now